www/about/policies/terms-of-service.md

```diff
@@ -35,94 +35,97 @@ or damage arising from your failure to keep your account secure.
-either party breaches the terms of the agreement (e.g. if the Team fails to
-provide the promised services).
+1. Terms for Project Owners
 
-  1. Gratipay is not obligated to provide refunds under any circumstance. Teams
-     are responsible for producing the Open Work they offer. Gratipay does not
-hold funds on behalf of Teams, and does not warrant or guarantee that Teams
-will provide the Open Work they promise.
+  1. A Project Owner can use the Service to receive payments from Givers for
+     Work provided by the Project's Collaborators. When you establish a Project
+on the Service, you agree that payments via the Project will be used
+exclusively to support the Work described in the Project's profile.
 
-1. Payroll Rules
+  1. Gratipay is not a party to any agreement between Project Owners and
+     Givers, and is not responsible to either the Project Owner or the Giver if
+either party breaches the terms of any such agreement (e.g. if the Project
+Owner fails to provide the promised goods or services).
 
-  1. Teams can use the Service's Payroll feature to automatically divide
-     incoming payments among Team Members, according to the allocation set by
-the Owner and/or Members.
+  1. A Project Owner can only begin receiving payments from Givers once the
+     Project Owner has provided a working withdrawal mechanism for receiving
+payments (e.g. bank account information).
 
-  1. Gratipay does not determine how a Team's Payroll is allocated and is not
-     responsible for how Teams internally distribute payments. The agreement
-between Team Members and Owners regarding the allocation of payments is solely
-between those parties.
+  1. Establishing a Project on Gratipay involves an application and approval
+     process. As part of this process, the Gratipay community will have the
+opportunity to publicly evaluate and provide commentary about the Project.
+However, all decisions regarding whether to accept, reject, suspend, remove, or
+take any other action regarding a Project belong solely to Gratipay.
 
-1. Rules for Gratipay Teams
+  1. Gratipay reserves the right to reject, suspend, or remove a Project at any
+     time, and for any reason.
 
-  1. Teams may accept payments on the Service only for Open Work, meaning that
-     they provide a clear path for any individual to voluntarily begin
-contributing to the Team's work, and to share in any revenue the Team
-generates.
+1. Terms for Collaborators
 
-  1. A Team can only begin receiving payments from Participants once it has
-     provided a working withdrawal mechanism for receiving payments (e.g. bank
-account information).
+  1. Project Owners can use the Service to distribute payments to
+     Collaborators.
 
-  1. Teams and their Open Work must be consistent with Gratipay's [Brand
-     Guidelines](http://inside.gratipay.com/big-picture/brand/) and Acceptable
-Use Policy.
+  1. When you become a Collaborator, you agree that payments to you by the
```

Comment:
Done in https://github.com/gratipay/gratipay.com/pull/4117/commits/ac4207ef6af1a5b7803450987ba71d70e3ee88a0.